### ISSUE TITLE
False negative when implementing a writable protocol attribute with a read-only (Final) version

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#3700]
```python
from typing import Protocol, Final

class Parent(Protocol):
    x: int

class Child(Parent):
    x: Final = 0 # This should generate an error but does not
```

**Comment by erictraut** [comment]
This will be included in the next release.

**Comment by erictraut** [comment]
This is included in pyright 1.1.261, which I just published. It will also be included in an upcoming release of pylance.
